apidoc/Doxyfile.in

```diff
@@ -1,4 +1,4 @@
-# Doxyfile 1.8.18
+# Doxyfile 1.8.16
```

Comment:
Are these changes in the Doxyfile relevant to this PR?
If not could you please remove them?

Reply:
Updated to 1.8.19